scripts/globals/events/harvest_festivals.lua

```diff
@@ -5,6 +5,231 @@ require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/utils")
 ------------------------------------
+events = events or {}
+
+events.harvest =
+{
+    skins =
+    {
+        orc = 1,
+        quadav = 2,
+        yagudo = 3,
+        shade = 4,
+        ghost = 5,
+        hound = 6,
+        skeleton = 7,
+        goblin = 8,
+    },
+    decorations =
+    {
+        [230] = { 17719828, 17719829, 17719830},
+        [231] = { 17723774, 17723775, 17723776, 17723777, 17723812, 17723811 },
+        [234] = { 17735939, 17735940, 17735941, 17735962, 17735961 },
+        [235] = { 17740139, 17740140, 17740141, 17740142, 17740143 },
+        [238] = { 17752504, 17752505, 17752506, 17752507, 17752548, 17752545 },
+        [241] = { 17764742, 17764743, 17764744},
```

Comment:
brace spacing inconsistent. some lines got spaces, some didn't.
@zach2good whats our convention now for spacing?